libs/api/metadata-converter/src/lib/iso19139/iso19139.converter.ts

```diff
@@ -339,6 +340,7 @@ export class Iso19139Converter extends BaseConverter<string> {
 
     fieldChanged('uniqueIdentifier') &&
       this.writers['uniqueIdentifier'](record, rootEl)
+    fieldChanged('reuseType') && this.writers['kind'](record, rootEl)
```

Comment:
This looks potentially confusing; maybe we can still have a "reuseType" writer and simply call the "kind" writer inside? At least we would have a homogeneous logic everywhere, even if it's an unnecessary step

Reply:
Ok I have added a new writer for `reuseType` that is calling the `kind` writer.